CODE_OF_CONDUCT.md

```diff
@@ -6,33 +6,34 @@ We are committed to making participation in this project a harassment-free exper
 
 Examples of behavior that contributes to creating a positive environment include:
 
-Using welcoming and inclusive language
-Being respectful of differing viewpoints and experiences
-Gracefully accepting constructive criticism
-Focusing on what is best for the community
-Showing empathy towards other community members
+-Using welcoming and inclusive language.  
+-Being respectful of differing viewpoints and experiences.  
+-Gracefully accepting constructive criticism.  
+-Focusing on what is best for the community.  
+-Showing empathy towards other community members.  
 
 Examples of unacceptable behavior by participants include:
 
-The use of sexualized language or imagery
-Personal attacks
-Trolling or insulting/derogatory comments
-Public or private harassment
-Publishing other's private information, such as physical or electronic addresses, without explicit permission
-Other unethical or unprofessional conduct
+-The use of sexualized language or imagery.  
+-Personal attacks.  
+-Trolling or insulting/derogatory comments.  
+-Public or private harassment.  
+-Publishing other's private information, such as physical or electronic addresses, without explicit permission. 
+-Other unethical or unprofessional conduct.  
+
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
 By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
 
 This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
 
-A working group of community members is committed to promptly addressing any reported issues. The working group is made up of pandas contributors and users. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the working group by e-mail (pandas-coc@googlegroups.com). Messages sent to this e-mail address will not be publicly visible but only to the working group members. The working group currently includes
+A working group of community members is committed to promptly addressing any reported issues. The working group is made up of this project's contributors and users. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the working group by e-mail (credit-prediction@googlegroups.com). Messages sent to this e-mail address will not be publicly visible but only to the working group members. The working group currently includes
```

Comment:
Does this email address exist ?

Reply:
Hi, just update the email address, please review it and see if it is good to merge, thanks!